### ISSUE TITLE
Test VPN connection

### ISSUE THREAD

**Issue by culmat** [baloise/CICD-pipeline#10]
see
https://github.com/baloise/CICD-pipeline-private/blob/main/private/VPN.md
https://github.com/baloise/CICD-pipeline-private/blob/main/private/Lab-Environment.md

Plan B : Have Azure Workspaces ( performance will be better through VPN )

**Comment by Solaflex** [comment]
Environment:
- Private Laptop with private Smartphone hotspot. No intereference from Baloise possible

Outcome:
- VPN Software asked for the trust of the certificate on first connection try
- Credentials were not working

**Comment by culmat** [comment]
I was able to connect on a private mac.
On the first connection attempt I had to enter the password many times ( the dialog kept popping up )
Now I can disconnect / reconnect entering the password only once.

![image](https://user-images.githubusercontent.com/638519/96872751-50d79680-1474-11eb-9b55-7f0cf0cd593f.png)

**Comment by culmat** [comment]
not working on DWP over Cisco Annyconnct ( home office situation )

**Comment by culmat** [comment]
lets have another test from inside Baloise Offices

**Comment by culmat** [comment]
@bhaskarchenchu : can you please add the IPs in https://github.com/baloise/CICD-pipeline-private/blob/main/private/Lab-Environment.md

**Comment by culmat** [comment]
the connection seems to be blocked by the baloise company proxy.
Investigating with the security team ...

**Comment by culmat** [comment]
won't fix, duplicated by #16
